static/src/javascripts/projects/common/modules/commercial/hosted-video.js

```diff
@@ -45,7 +44,9 @@ define([
     }
 
     function init() {
-        if (!$videoEl) {
+        $videoEl = $('.vjs-hosted__video');
+
+        if (!$videoEl || !$videoEl.length) {
```

Comment:
I'm pretty sure qwery always returns an array, so you could just check its length.